Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -214,6 +221,18 @@ struct PaymentSheetButtons: View {
     }
 }
 
+struct StaleView: View {
+    var body: some View {
+        Text("Stale")
+            .font(.subheadline.smallCaps().bold())
```

Comment:
textColor red would be nice :)

Reply:
Sure: <img width="276" alt="CleanShot 2023-06-05 at 13 36 40@2x" src="https://github.com/stripe/stripe-ios/assets/52758633/1c9b7fc1-099d-44c8-b779-51190f44d163">